docs/src/testing/contracts.md

```diff
@@ -105,7 +116,9 @@ If we called this function in a test, it would result in a failure.
 ```rust
 #[test]
 fn failing() {
-    // ...
+    let contract_address = deploy_contract('HelloStarknet');
```

Comment:
There is not such library function as `deploy_contract` yet, it is usually a helper function I write myself xD

Reply:
Maybe it would be worth to have such helper function in the docs?

Reply:
I mean here, in the guide, where users tend to follow along.

Reply:
I think it is enough to write

```cairo
// ...
let contract_address = deploy(prepared);
// the rest
```